<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSearchWin</class>
 <widget class="QWidget" name="CSearchWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1377</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>License Plate Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>60</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Leave Parking Lot Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Parking Lot Time:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>70</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>70</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>690</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>680</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Excel</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>1251</width>
     <height>481</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Num</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>License</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Enter Time</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Leave Time</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Enter Picture</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Leave Picture</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Paarking Price</string>
    </property>
   </column>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
